deployments/helm/nsm/charts/spire/registration.json

```diff
@@ -59,6 +79,36 @@
       ],
       "spiffe_id": "spiffe://test.com/nsmrs",
       "parent_id": "spiffe://test.com/spire-agent"
+    },
+    {
```

Comment:
we should make a utility to generate this file for our CI and not have any of the changes for authn of our control-plane or nse services in this repo.